### PR TITLE
Added signup and login forms

### PR DESCRIPTION
**Pull request by rochagbriel**
Login and sign-up features were implemented using React forms, applying validation and authentication mechanisms.

### CODE REVIEW

File: src/components/signup-view/signup-view.jsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                <input 
+                type='password'
+                value={password}
+                onChange={(e) => setPassword(e.target.value)}required
+                />
+            </label>
+            <label>
+                Email:
+                <input 
+                type='text'
```
Comment (Stanley-Okwii) [inline src/components/signup-view/signup-view.jsx]: You could use `type=email` for email validation here, remember the html5 validation?

File: src/components/login-view/login-view.jsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                if (data.user) {
+                    localStorage.setItem('user', JSON.stringify(data.user));
+                    localStorage.setItem('token', data.token);
+                    onLoggedIn(data.user, data.token);
+                } else {
+                    alert('No such user');
+                }
+            })
+            .catch((e) => {
+                console.log(JSON.stringify(data))
```
Comment (Stanley-Okwii) [inline src/components/login-view/login-view.jsx]: When your API encounters an error, the stack trace will be available in the `e` parameter.  Within your catch statement, variable `data` will only log request body i.e username and password. In this case, you can use `console.error("login error: ", e);`

File: src/components/login-view/login-view.jsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { useState } from 'react';
```
Comment (Stanley-Okwii) [inline src/components/login-view/login-view.jsx]: You should be consistent in your imports too. You shouldn't need to import twice from the same package.

**Review by Stanley-Okwii — APPROVED** [review]
Amazing work 🎉

File: src/components/signup-view/signup-view.jsx (hunk 1 above)
Comment (rochagbriel) [reply src/components/signup-view/signup-view.jsx]: done!Thanks

File: src/components/login-view/login-view.jsx (hunk 2 above)
Comment (rochagbriel) [reply src/components/login-view/login-view.jsx]: Done! 
My mistake! I did it to correct a mistake, but I forgot to change it again.😅

File: src/components/login-view/login-view.jsx (hunk 3 above)
Comment (rochagbriel) [reply src/components/login-view/login-view.jsx]: Done!

**Pull request merged**